android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -221,20 +221,20 @@ public void onHostDestroy() {
 
     // ====== CONNECTING ===========================================================================
 
-    public void connectToRoomWrapper(String accessToken) {
+    public void connectToRoomWrapper(String roomName, String accessToken) {
         /*
          * Check camera and microphone permissions. Needed in Android M.
          */
         Log.i("CustomTwilioVideoView", "Starting connect flow");
-        createLocalMedia(accessToken);
+        createLocalMedia(roomName, accessToken);
     }
 
-    public void connectToRoom(String accessToken) {
+    public void connectToRoom(String roomName, String accessToken) {
         /*
          * Create a VideoClient allowing you to connect to a Room
          */
         setAudioFocus(true);
-        ConnectOptions.Builder connectOptionsBuilder = new ConnectOptions.Builder(accessToken);
+        ConnectOptions.Builder connectOptionsBuilder = new ConnectOptions.Builder(accessToken).roomName(roomName);
```

Comment:
Do we need to check if roomName is null (which it might be since it's optional)?